pkg/policy/id_test.go

```diff
@@ -29,13 +29,15 @@ func TestIDIsValid(t *testing.T) {
 	}{
 		{ID("DenyEncryptionSt1"), true},
 		{ID(""), true},
+		{ID("abc-test"), true},
+		{ID("abc test"), true},
 	}
 
 	for i, testCase := range testCases {
 		result := testCase.id.IsValid()
 
 		if result != testCase.expectedResult {
-			t.Fatalf("case %v: result: expected: %v, got: %v\n", i+1, testCase.expectedResult, result)
+			t.Errorf("case %v: result: expected: %v, got: %v\n", i+1, testCase.expectedResult, result)
```

Comment:
Failing early is better than unnoticed `t.Errorf()`

Reply:
It's not unnoticed @balamurugana t.Errorf() simply continues to next test case. t.Fatalf is not correct.